xml/admin_rbd.xml

```diff
@@ -1368,6 +1368,7 @@ SCHEDULE TIME       IMAGE
    &ceph; supports write-back caching for RBD. To enable it, add
   </para>
 
+<!-- 2020-09-7 tbazant (ceph.conf): is the following still valid and usable? -->
```

Comment:
`ceph config set client rbd_cache true` *should* work, as long as this is actually still a valid config, which I didn't verified. Basically everything should work, except for some strange edge-cases, like very low-level stuff.